docs/endpoints.rst

```diff
@@ -9,15 +9,18 @@ or a Kubernetes cluster, for example.
 The endpoint requires outbound network connectivity. That is, it must be able to connect to
 funcX at `funcx.org <https://api2.funcx.org/v2>`_.
 
-To install the funcX endpoint agent software ::
+The funcX endpoint is available on pypi.org (and thus available via ``pip``).
+However, *we strongly recommend installing the funcX endpoint into an isolated virtual environment*.
+`Pipx <https://pypa.github.io/pipx/installation/>`_ automatically manages
+package-specific virtual environments for command line applications, so install funcX endpoint via::
 
-     $ python3 -m pip install funcx_endpoint
+     $ python3 -m pipx install funcx_endpoint
 
 .. note::
 
-   Please note that the funcx endpoint is only supported on Linux and MacOS.
+   Please note that the funcX endpoint is only supported on Linux and MacOS.
 
-After installing funcX endpoint, you be able to deploy new endpoints and interact
+After installing the funcX endpoint, you be able to deploy new endpoints and interact
```

Comment:
The grammar after the comma needs a little love ("... you be able to deploy ...").  Perhaps:

    After installing the funcX endpoint, use the ``funcx-endpoint`` command to manage local endpoints.

Reply:
ha, saw the missing 'the' and not the rest of the sentence to fix.